x-pack/plugins/apm/server/routes/settings/anomaly_detection.ts

```diff
@@ -6,6 +6,7 @@
 
 import * as t from 'io-ts';
 import Boom from 'boom';
+import { isValidPlatinumLicense } from '../../../common/service_map';
```

Comment:
Can we rename this to 
```suggestion
import { isActivePlatinumLicense } from '../../../common/service_map';
```